youtube_dl/extractor/rte.py

```diff
@@ -60,3 +56,71 @@ def _real_extract(self, url):
             'thumbnail': thumbnail,
             'duration': duration,
         }
+
+
+
+class RteRadioIE(InfoExtractor):
+    IE_NAME = 'rte.ie'
+    IE_DESC = 'Raidió Teilifís Éireann Radio'
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?rte\.ie/radio/utils/radioplayer/rteradioweb\.html
+        [#]!rii=(?:[0-9]+)(?:%3A|:)  # Channel ID
```

Comment:
Yeah, looks like I just don't have permission for this project. I was just hoping to avoid a bunch of pointless tests building up in the build queue. They take quite a while to run.